aten/src/ATen/native/native_functions.yaml

```diff
@@ -9369,6 +9369,19 @@
 - func: special_logit.out(Tensor self, float? eps=None, *, Tensor(a!) out) -> Tensor(a!)
   python_module: special
 
+- func: special_logsumexp(Tensor self, int[1] dim, bool keepdim=False) -> Tensor
+  python_module: special
+  device_check: NoCheck   # TensorIterator
+  variants: function
+  dispatch:
```

Comment:
Dispatch is not needed for alias. Plus explicitautograd is the correct key here.

Reply:
Should be fixed now. Thanks!